datafusion/tests/sql.rs

```diff
@@ -2966,6 +2966,45 @@ async fn query_count_distinct() -> Result<()> {
     Ok(())
 }
 
+#[tokio::test]
```

Comment:
We can also add it, but ignore the test?

Reply:
yeah, maybe that would be better...

If I am able to fix the problem in the next few days perhaps I can just incorporate this change into the proper fix 😆  

I'll leave this PR as a draft for now